rpcd/playbooks/roles/rpc_maas/defaults/main.yml

```diff
@@ -358,8 +358,9 @@ ceph_checks_list:
   - { name: "ceph_cluster_stats", group: "mons" }
 
 network_checks_list:
```

Comment:
This should just list the checks, not the interface.  The interface enumeration should be done in the task (network.yml) to generate a list of interfaces we wish to monitor.  If it's bonds that we wish to monitor this bash snippet can be used to register that variable.

    for interface in $(find /sys/class/net/ -maxdepth 1 -type l | rev | cut -d '/' -f 1 | rev); do if [[ $(cat /sys/class/net/"${interface}"/type) -eq 1 ]]; then if [[ $(grep DEVTYPE=bond /sys/class/net/"${interface}"/uevent) ]]; then echo "${interface}"; fi; fi; done

once that variable is registered it can then be used to create checks using `with_items` and the like.